### ISSUE TITLE
Website

### ISSUE THREAD

**Issue by crayzeigh** [devopsct/DevOpsDaysHartford2017#1]
So this is the main case: Do we want to have a website separate from the devopsdays.org main site?

Examples of Yes:
* [DevOpsDays Rockies](http://devopsdaysrox.org)
* [DevOpsDays Austin](http://www.devopsdaysaustin.com/)

Examples of No:
Almost everyone else

## Arguments in favor:
* Faster Update response time 
* More local branding
* Looks pretty

## Arguments Against:
* Has to be maintained by one of us (one of us)
* More complication and maintaining information in two places instead of one.

## To sum up

I basically don't want to maintain it. Otherwise if someone wants to commit to maintaining a site separately for DevOps Days Hartford (or other city as future determines) than I'm fine with it and will contribute. I don't (personally) want to deal with the logistics.

**Comment by davejlong** [comment]
👍 towards no website.

**Comment by mego22** [comment]
I did a search last night and more DevOps Days are putting sites up. I don't mind taking the reigns on this and keeping it up-to-date.
